account_analytic_compassion/__manifest__.py

```diff
@@ -34,12 +34,15 @@
     'license': 'AGPL-3',
     'website': 'http://www.compassion.ch',
     'depends': [
+        'account_analytic_attribution',
```

Comment:
account_analytic_attribution is a module that we want to remove. Why did you need to add this dependency? I think it should be avoided.